src/com/etendoerp/dependencymanager/process/AddDependency.java

```diff
@@ -89,18 +95,55 @@ protected JSONObject execute(Map<String, Object> parameters, String data) {
       errorMessage.put(MESSAGE, message);
       OBDal.getInstance().flush();
       errorMessage.put("refreshParent", true);
-    } catch (JSONException jsone) {
+    } catch (JSONException json) {
       try {
         JSONObject message = new JSONObject();
         message.put(SEVERITY, "error");
         message.put(TITLE, "Error");
         errorMessage.put(MESSAGE, message);
```

Comment:
You are creating a new JSONObject for the message, but you are not populating it; therefore the message will be empty

Reply:
Are you sure?  both JSONObject variables called  message were seting severity and title and were asigned into return variable errorMessage

Reply:
My bad, totally oversaw it 🙈